crates/sui/src/client_commands.rs

```diff
@@ -644,6 +645,13 @@ pub enum SuiClientCommands {
         #[arg(long, short)]
         terminate_early: bool,
     },
+
+    /// eth light-client subcommands.
```

Comment:
```suggestion
    /// Ethereum light-client subcommands.
```

Start comments with capital letter, using full names.